### ISSUE TITLE
Install Instructions updated dependencies

### ISSUE THREAD

**Issue by Vash63** [cschied/q2vkpt#28]
I was initially a bit turned off seeing the install guide ask for a bunch of old deps that are no longer available within Arch's official repos. Thankfully after attempting to blindly build the project I found that none of them were needed. This is the list of libraries and their respective packages as required in Arch - I'm not sure what their Ubuntu equivalents are so I'll refrain from trying to edit your install guide directly, but it would be helpful for people trying to build this project to have a summary of current dependencies.

```
/usr/lib/libvulkan.so.1 is owned by vulkan-icd-loader 1.1.96+3009+32d33e965-1
/usr/lib/libpng16.so.16 is owned by libpng 1.6.36-1
/usr/lib/libjpeg.so.8 is owned by libjpeg-turbo 2.0.1-1
/usr/lib/libz.so.1 is owned by zlib 1:1.2.11-3
/usr/lib/libSDL2-2.0.so.0 is owned by sdl2 2.0.9-1
/usr/lib/libm.so.6 is owned by glibc 2.28-5
/usr/lib/libdl.so.2 is owned by glibc 2.28-5
/usr/lib/libpthread.so.0 is owned by glibc 2.28-5
/usr/lib/libc.so.6 is owned by glibc 2.28-5
/usr/lib/ld-linux-x86-64.so.2 is owned by glibc 2.28-5
/usr/lib/librt.so.1 is owned by glibc 2.28-5
```

In summary:
glibc
zlib
sdl2
libjpeg-turbo
libpng
vulkan-icd-loader
